api/http/Server.go

```diff
@@ -30,6 +31,12 @@ func (s *Server) Serve(wait *sync.WaitGroup) error {
 		return err
 	}
 
+	handler := cors.New(cors.Options{
+		AllowedMethods: []string{"GET", "POST", "PATCH", "PUT", "DELETE"},
+		AllowedHeaders: []string{"Authorization", "Content-Type"},
+		MaxAge:         300,
+	}).Handler(mux)
```

Comment:
I think we also need to set `AllowedCredentials: true` to allow for  user credentials like cookies, HTTP authentication or client side SSL certificates, but not sure

Reply:
resolving this, as AllowCredentials seems to have been added.